spec/system/planning_applications/clone_spec.rb

```diff
@@ -117,7 +117,7 @@
         expect(page).to have_current_path("/planning_applications/#{cloned_planning_application.id}")
         expect(page).to have_content("Planning application was successfully cloned")
 
-        expect(JSON.parse(planning_application.params_v2)).to eq(JSON.parse(cloned_planning_application.params_v2))
+        expect(JSON.parse(planning_application.params_v2)).to eq(cloned_planning_application.params_v2)
```

Comment:
This seems to be messed up test setup - the params are a string instead of being a hash